### DISCUSSION TITLE
Manque les acteurs qui n'ont pas de film dans ActorInPGTest

### DISCUSSION THREAD

**Discussion by Greenkiller21** [amt-classroom/amt-classroom.github.io#8] [General]
Dans le test actuel il manque les personnes qui n'ont pas de films. Et donc le test qui regarde s'il y'a bel et bien 199 est faux, car NOLTE SALMA n'a pas réalisé de films et actuellement cela n'est pas pris en compte dans le nombre total (qui devrait donc être 200).

**Comment by Perdjesk** [comment]
Les résultats doivent contenir uniquement les `actor` qui ont au moins un film associé au `rating` PG.
